eva-server/src/main/java/uk/ac/ebi/eva/server/ws/RegionWSServerV2.java

```diff
@@ -87,11 +91,19 @@ public ResponseEntity getVariantsByRegion(@PathVariable("regionId") String regio
                                                       annotationVepVersion,
                                               @RequestParam(name = "annot-vep-cache-version", required = false) String
                                                       annotationVepCacheVersion,
+                                              @RequestParam(value = "page", required = false) Integer page,
+                                              @RequestParam(value = "size", required = false) Integer size,
                                               HttpServletResponse response,
                                               @ApiIgnore HttpServletRequest request)
             throws IllegalArgumentException {
         checkParameters(annotationVepVersion, annotationVepCacheVersion, species);
 
+        if (page == null) {
```

Comment:
You can set the deafult value in the `RequestParam` annotation [link](https://docs.spring.io/spring/docs/4.3.14.RELEASE/javadoc-api/org/springframework/web/bind/annotation/RequestParam.html)

Reply:
Fixed :)

Reply:
Delete the constants `DEFAULT_PAGE_SIZE` and `DEFAULT_PAGE` if they will not be used

Reply:
Deleted :)